02week/ticTacToe.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const assert = require('assert');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+let board = [
+  [' ', ' ', ' '],
+  [' ', ' ', ' '],
+  [' ', ' ', ' ']
+];
+
+let playerTurn = 'X';
+
+function printBoard() {
+  console.log('   0  1  2');
+  console.log('0 ' + board[0].join(' | '));
+  console.log('  ---------');
+  console.log('1 ' + board[1].join(' | '));
+  console.log('  ---------');
+  console.log('2 ' + board[2].join(' | '));
+}
+
+// Should return true if player won on any row
+function horizontalWin() {
+  for (let i=0; i <= 2; i++) {
+    for (let a=0; a <=2; a++) {
+      if (board[i][a] === "X" && board[i][a] === "X" && board[i][a] === "X") {
+        return true;
+      }
+    }
+  }
+}
+horizontalWin();
+
+// Should return true if player won on any column
+function verticalWin() {
+  for (let i=0; i <= 2; i++) {
+    for (let a=0; a <=2; a++) {
+      if (board [a][i] === "X" && board[a][i] === "X" && board[a][i]) {
+        return true;
+      }
+    }
+  }
+}
+verticalWin();
+
+// Should return true if player won on any diagonal
+function diagonalWin() {
+  for (let i=0; i <= 2; i++) {
+    if (board[i][i] === "X") {
+
+    }
+  }
+}
+
+// Should return true if any above functions return true
+function checkForWin() {
+  if (horizontalWin() === true) {
+    return true;
```

Comment:
you dont need to explicitly check  that the result is equals to `true` since you could just evaluate the result itself.
``` if(horizontalWin()) {
...
```